src/styles/card.scss

```diff
@@ -0,0 +1,126 @@
+* {
+  box-sizing: border-box;
+}
+
+.cards {
+  --column-count: 1;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  row-gap: 46px;
+  column-gap: 48px;
+  justify-content: center;
+  padding: 50px 40px;
+}
+
+@media (max-width: 488px) {
+  .cards {
+    --column-count: 1;
+  }
+}
+
+@media (min-width: 488px) {
+  .cards {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .cards {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .cards {
+    --column-count: 4;
+  }
+}
+
+.card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $white;
```

Comment:
card should have padding for all sides to avoid margin-left/margin-right for inner elements